### ISSUE TITLE
[Bug]: Palworld start server after last update

### ISSUE THREAD

**Issue by infernvs** [GameServerManagers/LinuxGSM#4550]
### User story

When using `./pwserver start` gives an error

### Game

Palworld

### Linux distro

Ubuntu 22.04

### Command

command: start

### Further information

after this last update `13920882` when starting the server gives the error that PalServer-Linux-Test is not found, inside the folder there is only `ls
PalServer-Linux-Shipping  PalServer-Linux-Shipping.sym`

### Relevant log output

```shell
./pwserver st
[ FAIL ] Starting pwserver: executable was not found
* /home/palworld/serverfiles/Pal/Binaries/Linux//PalServer-Linux-Test
```

**Comment by Deathicated** [comment]
Same issue

**Comment by Ramoth** [comment]
> adding `executable="./PalServer-Linux-Shipping"` to the _pwserver.cfg_ on `~/lgsm/config-lgsm/pwserver` seems to fix the issue

Can confirm that this fixes the issue

**Comment by lblanchardiii** [comment]
Just found this, I took another route to fix the problem. Not sure which is more correct, but I just created two symbolic links that point to the old "Test" version.

I just navigated to the /user/serverfiles/Pal/Binaries/Linux/ directory and typed these commands to create symbolic links for each one of those files. Server started right up.

ln -s PalServer-Linux-Shipping.sym PalServer-Linux-Test.sym
ln -s PalServer-Linux-Shipping PalServer-Linux-Test

I assume a future LGSM script update will correct the problem, which means I wont have to do anything when it does. The symbolic links take up literally no space and I can delete them later if they end up not being needed.
